src/DirectionProvider.jsx

```diff
@@ -48,7 +51,7 @@ export default class DirectionProvider extends React.Component {
     const { children, direction, inline } = this.props;
     const Tag = inline ? 'span' : 'div';
     return (
-      <Tag dir={direction}>
+      <Tag dir={direction || undefined}>
```

Comment:
On html tags, this may or may not be the same; it may need to be `{...(direction && { direction }) }` (test in a browser to make sure)